packages/engine/Source/Scene/OctahedralProjectedCubeMap.js

```diff
@@ -337,11 +337,10 @@ OctahedralProjectedCubeMap.prototype.update = function (frameState) {
     },
   });
 
-  // We only need up to 6 mip levels to avoid artifacts.
-  const length = Math.min(cubeMapBuffers.length, 6);
-  this._maximumMipmapLevel = length - 1;
-  const cubeMaps = (this._cubeMaps = new Array(length));
-  const mipTextures = (this._mipTextures = new Array(length));
+  const mipLevels = cubeMapBuffers.length;
```

Comment:
Would it be worth capping this to a higher number than 6?

Reply:
I tried a few values with mixed results. My choice of no cap was based on 2 factors:
1. The higher-numbered LODs are the lower-resolution ones, so including them adds very little memory overhead.
2. If a user tries the same environment map at different resolutions, they would expect to see differences only in the high-res details of the reflections. However, if we cap the max LOD, different input resolutions will result in a different mip being used for the background low-res part of the reflection.